elb_automation/meaoelb/sumo_oregon_b.py

```diff
@@ -0,0 +1,27 @@
+from meaoelb.oregon_b import elb_tool
+
+# ### SUMO Stage
+sumo_stage = elb_tool.define_elb_http(
+    service_namespace='sumo-stage',
+    service_name='sumo-nodeport',
+    ssl_arn='arn:aws:acm:us-west-2:236517346949:certificate/192b6409-996e-46ac-a3d9-c78a69670dae')
+sumo_stage.elb_config.health_check.target_path = '/healthz/'
+sumo_stage.elb_config.name = 'sumo-stage-b'
+
+# ### SUMO prod
+sumo_prod = elb_tool.define_elb_http(
```

Comment:
this one is tricky because it's bound to 2 ASGs, and with the `define_elb_http` method we'll only be bound to 1.
I wonder if it's easier to skip the convenience methods and define the entire elb, following by a call to the asg bind code:

- https://github.com/mozmeao/infra/blob/master/elb_automation/meaoelb/ap_northeast_1.py#L97-L147
- https://github.com/mozmeao/infra/blob/master/elb_automation/meaoelb/ap_northeast_1.py#L524
- https://github.com/mozmeao/infra/blob/master/elb_automation/meaoelb/elb_ctx.py#L147

Reply:
This was for the oregon-b specific ELB; the ELB that was bound to 2 ASGs was created manually in the console.